### PR TITLE
[OneToolchain] Returns the output value as it is

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit returns the output value as it is.
This allows the log to be displayed and the output to be handled.

ONE-vscode-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

~~fix this bug: https://github.com/Samsung/ONE-vscode/pull/1333#issuecomment-1268162214, #1335~~

### CODE REVIEW

**Review by Psalmist-KIM — APPROVED** [review]

**Comment by lettersto** [comment]
Is this to see logs when prerequisites fails?

**Comment by jyoungyun** [comment]
> Is this to see logs when prerequisites fails?

File: script/prerequisitesForGetToolchains.sh [hunk 1]
```diff
@@ -1,8 +1,8 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (lettersto) [inline script/prerequisitesForGetToolchains.sh:1]: I see the change in shebang. So, do we have to change `sh` to `bash` in `prerequistesForGetToolchains` too?

Comment (jyoungyun) [reply script/prerequisitesForGetToolchains.sh:1]: Our project usually uses bash shell. So I changed it too. :)

**Comment by jyoungyun** [comment]
The #1335 error does not occur in my test environment. However, printing the log will help you determine what happened. So I want to merge this PR.

File: script/prerequisitesForGetToolchains.sh (hunk 1 above)
Comment (lettersto) [reply script/prerequisitesForGetToolchains.sh:1]: I see :) Thank you!

**Review by lettersto — APPROVED** [review]

**Review by llFreetimell — APPROVED** [review]

File: script/prerequisitesForGetToolchains.sh [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
 # This shell script add PPA to get onecc-docker package for ONEToolchain.
 
 PPA_NAME="ppa:one-compiler/onecc-docker"
 
-add-apt-repository -y ${PPA_NAME} > /dev/null 2>&1 &&
-        apt-get update > /dev/null 2>&1
+add-apt-repository -y ${PPA_NAME}
+apt-get update -yqq
```
Comment (llFreetimell) [inline script/prerequisitesForGetToolchains.sh:8]: AFAIK, `-yqq` is not related with printing output value, isn't it?

Comment (jyoungyun) [reply script/prerequisitesForGetToolchains.sh:8]: My intention is to print the `add-apt-repository` logs. I thought the `apt-get update` log was too long and unnecessary.

Comment (llFreetimell) [reply script/prerequisitesForGetToolchains.sh:8]: OK, I see it!

Comment (jyoungyun) [reply script/prerequisitesForGetToolchains.sh:8]: Ah, `-yqq` means that no output except for errors. It does not related to output value. :)

**Pull request merged**